int32 mode
int32 duration
int32 motor1_move
int32 motor2_move
int32 motor1_torque
int32 motor2_torque
float32 PF
float32 EV
float32 CPU0
float32 CPU1
float32 CPU2
float32 CPU3
float32 t
